canvas_modules/common-canvas/src/common-properties/controls/multiselect/multiselect.jsx

```diff
@@ -167,6 +167,7 @@ class MultiSelectControl extends React.Component {
 					titleText={this.props.tableControl ? null : this.props.controlItem}
 					helperText={this.props.control.helperText}
 					readOnly={this.props.readOnly}
+					aria-label={!this.props.control.labelVisible && this.props.control?.label?.text}
```

Comment:
`aria-label` is deprecated. This component uses `titleText` for aria attributes
![Screenshot 2025-02-28 at 10 15 22 AM](https://github.com/user-attachments/assets/f0f8a601-b41a-403e-b128-f9153c77bfd8)

Aria-label isn't showing up in DOM after setting `label_visible: false`.

Reply:
Thanks. I have removed this as this is not supported and also there is no violation showing up related to this.